Makefile

```diff
@@ -87,6 +87,18 @@ $(HEADS_GAWK): $(build)/$(gawk_dir)/.configured
 		$(VERBOSE_REDIRECT)
 endif
 
+# Some things want usernames, we use the current checkout
+# so that they are reproducible
+GIT_HASH		:= $(shell git rev-parse HEAD)
+GIT_HASH_SHORT	:= $(shell git rev-parse --short HEAD)
+GIT_BRANCH  	:= $(shell git branch --show-current)
+GIT_TAG     	:= $(shell git describe --abbrev=0 --tags)
```

Comment:
feel like 'git describe --abbrev=1 --tags --dirty' would be better here, like coreboot uses, since the number of commits on top of the tag is useful. `v0.2.0-912-g46465-dirty` is better than `v0.2.0-g46465` IMO

This also simplifies things elsewhere since we don't need to separately include the hash